tests/health.rs

```diff
@@ -32,7 +32,10 @@ async fn health_server() {
     });
     t.run_server(
         server_config,
-        None,
+        Some(quilkin::Proxy {
+            qcmp_port: 9094,
```

Comment:
If it's unused shouldn't it be `0`? that way it wouldn't ever conflict bound ports.
```suggestion
            qcmp_port: 0,
```